Cython/Compiler/ModuleNode.py

```diff
@@ -781,6 +781,7 @@ def generate_module_preamble(self, env, options, cimported_modules, metadata, co
         code.putln('#define __Pyx_PyObject_FromString __Pyx_Py%s_FromString' % c_string_func_name)
         code.putln('#define __Pyx_PyObject_FromStringAndSize __Pyx_Py%s_FromStringAndSize' % c_string_func_name)
         code.put(UtilityCode.load_as_string("TypeConversions", "TypeConversion.c")[0])
+        env.use_utility_code(UtilityCode.load_cached("FormatTypeName", "ObjectHandling.c"))
```

Comment:
It seems you already added this as a dependency in lots of places. Then we don't need this here.

Reply:
It's used in the `TypeConversion.c::TypeConversions` fragment. But for some reasons the `//@requires` seems to have no effect when it is loaded with `UtilityCode.load_as_string` so I manually called the `load_cached` here...

Reply:
In that case, you don't need it anywhere else, because it's already included globally.
In the end, it's such a basic thing that it probably _is_ worth always including it. The only reason why it's not just part of the compatibility section in `ModuleSetupCode.c` is that it uses Python string constants.

Remove it as a dependency everywhere else and then just include it here.

Reply:
fair enough.